CMSgov/design-system
# Refactor help drawer to support sticky header/footer.

## Summary
[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1137)
Implement sticky header and footer props on HelpDrawer component.

### Added
Added two new props: `isHeaderSticky` and `isFooterSticky` to support sticky positioning of those two elements.

### Changed
To accommodate the optional sticky positioning, HelpDrawer needed to be refactored. 

New implementation was tested in Safari, Chrome and Firefox (Mac) and no changes were noticed between local and prod components. Will test on Edge and IE after preview url is generated.

### Removed
A bunch of markup and CSS classes.

## How to test
`WNMGDS-1137/helpdrawer-sticky-props` will be located at [test url](http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1137/helpdrawer-sticky-props/components/help-drawer/)

To see sticky props in action, you can either turn the props on/off via the React Component tools in Chrome's browser dev tools, or add/remove the `ds-c-help-drawer__header--is-sticky` and `ds-c-help-drawer__footer--is-sticky` classes.
